samples/meetings-transcription/csharp/README.md

```diff
@@ -0,0 +1,147 @@
+﻿---
+page_type: sample
```

Comment:
Looks like ReadME UI is broken:
![image](https://user-images.githubusercontent.com/59052547/173110004-7b4e9015-de29-435a-aa63-68a0266d3d79.png)

Reply:
Resolved